db/migrate/20240902111455_change_adopter_foster_account_null_on_adopter_applications.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeAdopterFosterAccountNullOnAdopterApplications < ActiveRecord::Migration[7.1]
```

Comment:
Do you think we should replace this index with form_submission_id instead of completely removing it?

Reply:
@kasugaijin Doesn't this index exist already? https://github.com/rubyforgood/pet-rescue/blob/a86d748b5462c6ebd8084e3a59390e3da05b4f8a/db/schema.rb#L54

Reply:
Ah you are right! Was added in Ken's PR. Thanks for that.